themes/default/content/product/pulumi-deployments.md

```diff
@@ -9,6 +9,20 @@ overview:
     description: |
         [Pulumi Deployments](/docs/pulumi-cloud/deployments/) is a new feature that automates the execution of your Pulumi programs in a secure, hosted environment. Deploy any stack with a click of a button, `git push`, or API call. Available in preview today.
 
+quotes:
+    items:
+        - company: mighty
+          name: Aaron Gibralter
+          name_title: Head of Engineering at Mighty
+          quote: |
+            “Pulumi allows every engineer to build and update infrastructure instead of only our infrastructure team, which enables us to ship faster than if we were using a domain-specific language. With Pulumi Deployments, we could easily set up a standardized CI/CD infrastructure workflow for every engineer that immediately makes them productive. We’re eager to adopt Review Stacks because it will automate the work of managing our own ephemeral infrastructure and make our lives easier.”
```

Comment:
```suggestion
            “Pulumi allows every engineer to build and update infrastructure instead of only our infrastructure team. With Pulumi Deployments, we could easily set up a standardized CI/CD infrastructure workflow for every engineer that immediately makes them productive.”
```